bit-ignore.bazelrc

```diff
@@ -0,0 +1 @@
l/sample/res/values-ja,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-en-rCA,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/layout,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/mipmap-xxhdpi,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-night,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-fr-rCA,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-en-rGB,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-en-rAU,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/mipmap-xhdpi,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/values-es,examples/anvil/app/src/main/java/com/squareup/anvil/sample/res/mipmap-anydpi-v26,examples/anvil/library,examples/anvil/library/src,examples/anvil/library/src/main,examples/anvil/library/src/main/java,examples/anvil/library/src/main/java/com,examples/anvil/library/src/main/java/com/squareup,examples/anvil/library/src/main/java/com/squareup/anvil,examples/anvil/library/src/main/java/com/squareup/anvil/sample,examples/anvil/library/src/main/java/com/squareup/anvil/sample/father,examples/anvil/library/src/main/java/com/squareup/anvil/sample/mother,examples/anvil/third_party,examples/anvil/repro,examples/anvil/repro/src,examples/anvil/repro/src/main,examples/anvil/repro/src/main/java,examples/anvil/repro/src/main/java/com,examples/anvil/repro/src/main/java/com/repro,examples/anvil/repro/src/main/java/com/repro/lib,examples/anvil/scopes,examples/anvil/scopes/src,examples/anvil/scopes/src/main,examples/anvil/scopes/src/main/java,examples/anvil/scopes/src/main/java/com,examples/anvil/scopes/src/main/java/com/squareup,examples/anvil/scopes/src/main/java/com/squareup/scopes,examples/associates,examples/associates/projects,examples/associates/projects/core,examples/associates/projects/core/impl,examples/associates/projects/core/impl/src,examples/associates/projects/core/impl/src/test,examples/associates/projects/core/impl/src/test/kotlin,examples/associates/projects/core/impl/src/test/kotlin/core,examples/associates/projects/core/impl/src/test/kotlin/core/impl,examples/associates/projects/core/impl/src/main,examples/associates/projects/core/impl/src/main/kotlin,examples/associates/projects/core/impl/src/main/kotlin/core,examples/associates/projects/core/impl/src/main/kotlin/core/impl,examples/associates/projects/core/api,examples/associates/projects/core/api/src,examples/associates/projects/core/api/src/test,examples/associates/projects/core/api/src/test/kotlin,examples/associates/projects/core/api/src/test/kotlin/core,examples/associates/projects/core/api/src/test/kotlin/core/api,examples/associates/projects/core/api/src/main,examples/associates/projects/core/api/src/main/kotlin,examples/associates/projects/core/api/src/main/kotlin/core,examples/associates/projects/core/api/src/main/kotlin/core/api
```

Comment:
Is it possible to just ignore the entire directory here instead of each individual package?

Reply:
Unfortunately, no.

I tried that, but `--deleted_packages` only removes each package, not the subpackages.